next.clayui.com/content/docs/migration/migrate-the-clay-components-from-v2-to-v3.md

```diff
@@ -796,14 +893,16 @@ ClayDropDown also exports `<ClayDropDown.Menu>` which can be used independently
 
 See `ClayDropDown` in our storybook for more compositions and examples of use.
 
+Also, it is offered as a High-Level component called [DropDownWithItems](/docs/components/drop-down.html#claydropdownwithitems).
```

Comment:
We go back and forth between capitalizing and not capitalizing "high-level", I think it'd be best to not capitalize and use lower case for all instances.